axelrod/fingerprint.py

```diff
@@ -53,110 +54,175 @@ def create_points(step, progress_bar=True):
     return points
 
 
-class AshlockFingerprint():
-    def __init__(self, strategy, probe=axl.TitForTat):
-        """
-        Parameters
-        ----------
-        strategy : class or instance
-            A class that must be descended from axelrod.Player or an instance of
-            axelrod.Player.
-        probe : class or instance
-            A class that must be descended from axelrod.Player or an instance of
-            axelrod.Player.
-            Default: Tit For Tat
-        """
-        self.strategy = strategy
-        self.probe = probe
+def create_jossann(point: Point, probe: Any) -> Player:
+    """Creates a JossAnn probe player that matches the Point.
 
-    @staticmethod
-    def create_jossann(point, probe):
-        """Creates a JossAnn probe player that matches the Point.
+    If the coordinates of point sums to more than 1 the parameters are
+    flipped and subtracted from 1 to give meaningful probabilities. We also
+    use the Dual of the probe. This is outlined further in [Ashlock2010]_.
 
-        If the coordinates of point sums to more than 1 the parameters are
-        flipped and subtracted from 1 to give meaningful probabilities. We also
-        use the Dual of the probe. This is outlined further in [Ashlock2010]_.
+    Parameters
+    ----------
+    point : Point
+    probe : class
+        A class that must be descended from axelrod.strategies
```

Comment:
since this takes either a class or an instance, perhaps it should mirror the docstring for `AshlockFingerprint.__init__`, 
which is

```
probe : class or instance
            A class that must be descended from axelrod.Player or an instance of
            axelrod.Player.
```